packages/common/src/eips/2315.json

```diff
@@ -3,7 +3,7 @@
   "number": 2315,
   "comment": "Simple subroutines for the EVM",
   "url": "https://eips.ethereum.org/EIPS/eip-2315",
-  "status": "Draft",
+  "status": "EXPERIMENTAL",
```

Comment:
Ah, no, *this* status here is really the EIP status. This `EXPERIMENTAL` is only [used](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/vm/src/index.ts#L46) in the VM to *then* indicate the status of the implementation.

I guess if we update here to `EXPERIMENTAL` and mix this with the EIP states it get's chaotic. That said, this `status` field is also only used internally, I added this when setting up the `Common` class with the above semantics in mind. Not sure if it's very helpful and we generally want to keep or not.

We might want to discuss separately. In doubt we just repeat redundant information here, which will likely always get outdated again and again. I guess if we decide we could even "just remove" without the fear of any breaking-change consequences.

Nevertheless: for now will update here back to `Draft` to keep things consistent.

Reply:
But, to add some own opinion on that: I am currently also undecided (to remove or not), this might also be some useful helper on the status of EIPs we have implemented (otherwise one would likely not always look this up manually).